models/repository/repository.go

```diff
@@ -11,18 +11,18 @@ type Repository interface {
 }
 
 type RepositoryStruct struct {
-	Id           int64
-	RepositoryId int64
+	ID           int64
+	RepositoryID int64
 	Owner        sql.NullString
 	Name         sql.NullString
 	database     db.DB
 }
 
-func NewRepository(id int64, repositoryId int64, owner string, name string) *RepositoryStruct {
-	if repositoryId <= 0 {
+func NewRepository(id int64, repositoryID int64, owner string, name string) *RepositoryStruct {
```

Comment:
__[golint]__
* [golint] exported function NewRepository should have comment or be unexported